### DISCUSSION TITLE
"ctrl+opt+enter" prints [27;7;13~

### DISCUSSION THREAD

**Discussion by anatskiy** [ghostty-org/ghostty#3725] [Q&A]
I have a global shortcut that maximizes a window in macOS (ctrl+opt+enter).

In Ghostty, however, it prints the text `[27;7;13~` instead. I tried to fix the issue by creating a new keybinding to send a correct CSI sequence (according to [this](https://www.leonerd.org.uk/hacks/fixterms/)) but that didn't work.
```
keybind = ctrl+opt+enter=csi:13;7u
```
Now, Ghostty prints `[13;7u`.

I also tried all combinations of `macos-option-as-alt` which didn't help either.

I checked the CSI sequence printed by iTerm when pressing ctrl+opt+enter:
<img width="977" alt="Screenshot" src="https://github.com/user-attachments/assets/b31a7901-3458-4af2-bbb1-d4298de46446" />

Is there a way to fix this issue apart from changing the global shortcut (which I'm quite used to)?

**Comment by 00-kat** [comment]
Happens with `ctrl+alt+enter` too on Linux.

![image](https://github.com/user-attachments/assets/9f180a40-300d-465e-85ac-8182dcdc4bb2)
